src/test/java/de/mediathekview/mserver/crawler/srf/parser/SrfSendungenOverviewJsonDeserializerTest.java

```diff
@@ -16,17 +17,20 @@
 
 @RunWith(Parameterized.class)
 public class SrfSendungenOverviewJsonDeserializerTest {
+  
+  private static final String MONTH_YEAR = LocalDateTime.now().getMonthValue() + "-" + LocalDateTime.now().getYear();
+  
   @Parameterized.Parameters
     public static Collection<Object[]> data() {
         return Arrays.asList(new Object[][] { 
           { "/srf/srf_overview_page_data_section.json", 
             new CrawlerUrlDTO[] { 
-              new CrawlerUrlDTO("https://www.srf.ch/play/v2/tv/show/6fd27ab0-d10f-450f-aaa9-836f1cac97bd/latestEpisodes/tillMonth/11-2017?numberOfEpisodes=10&layout=json"), 
-              new CrawlerUrlDTO("https://www.srf.ch/play/v2/tv/show/c38cc259-b5cd-4ac1-b901-e3fddd901a3d/latestEpisodes/tillMonth/11-2017?numberOfEpisodes=10&layout=json"),
-              new CrawlerUrlDTO("https://www.srf.ch/play/v2/tv/show/420426e5-4e4d-4ba7-ab44-09a4e17c13bf/latestEpisodes/tillMonth/11-2017?numberOfEpisodes=10&layout=json"),
-              new CrawlerUrlDTO("https://www.srf.ch/play/v2/tv/show/c5a89422-4580-0001-4f24-1889dc30d730/latestEpisodes/tillMonth/11-2017?numberOfEpisodes=10&layout=json"),
-              new CrawlerUrlDTO("https://www.srf.ch/play/v2/tv/show/c5e431c3-ab90-0001-3228-16001350159c/latestEpisodes/tillMonth/11-2017?numberOfEpisodes=10&layout=json"),
-              new CrawlerUrlDTO("https://www.srf.ch/play/v2/tv/show/0a7932df-dea7-4d8a-bd35-bba2fe2798b5/latestEpisodes/tillMonth/11-2017?numberOfEpisodes=10&layout=json")
+              new CrawlerUrlDTO("https://www.srf.ch/play/v2/tv/show/6fd27ab0-d10f-450f-aaa9-836f1cac97bd/latestEpisodes?numberOfEpisodes=100&tillMonth=" + MONTH_YEAR + "&layout=json"), 
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "&layout=json" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)